priv/repo/migrations/20220701115243_create_peers_metrics.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Repo.Migrations.CreatePeersMetrics do
```

Comment:
The migrations from these modules don't make sense separately. They should be introduced with one transaction, thus applied by a single migration